### ISSUE TITLE
Overrided default ember g component generation

### ISSUE THREAD

**Issue by job13er** [ciena-frost/ember-cli-frost-blueprints#9]
I saw in the latest versions, 0.32.2, that ember g component is been overrided but I found this PR #170 where there is an indication to use frost-component instead to avoid override default component blueprint

Is this suppose to happen in that way?

This was actually originally raised by @albertonavarro087 [here](https://github.com/ciena-frost/ember-frost-core/issues/272)

Once we remove the blueprints from `ember-frost-core` and have them just here, this may be moot.

**Comment by job13er** [comment]
@sglanzer 

With the blueprints moving to this addon, could we just, in the next major release, have `ember-frost-core` and maybe `ember-frost-test` remove this from their `dependencies` and then the opt-out would just be not installing this addon? Seems easier than adding a flag for each of our generators to do the default behavior.

**Comment by sglanzer-fire** [comment]
@job13er yeah, that makes sense to me - they're really about developing components/test the "Frost" way, so we can document that for contributors, but consumers don't need to worry about it (other than Ciena consumers)

**Comment by job13er** [comment]
I created https://github.com/ciena-frost/ember-frost-core/issues/459 and https://github.com/ciena-frost/ember-frost-test/issues/31 to remove the dependencies in the 2.0 LTS

**Comment by job13er** [comment]
We really need that mechanism for having a central `CONTRIBUTING.md` file we can populate into all our repos... seems like that would be a good place for it. I'll create a repo in bitbucket to host that stuff, even if we only manually copy them over for now.
